Code.gs

```diff
@@ -46,6 +50,10 @@ function GenerateForms() {
       managerEmail = rowValues[3] + "@redhat.com";
       reviewers = rowValues[4].split(",").map(function(item) { return item.trim() + "@redhat.com" })
 
+      if (managerEmail != filterOnlyDirectsOf) {
```

Comment:
not sure I understood the logic of this?

Reply:
It allows to have a single spreadsheet with multiple managers sending the emails only for their relevant associates.

Reply:
but it would prevent the previous use case in which you were able to send for multiple groups, preventing the need for each manager to run the script on their own?

Reply:
@yanjoseph can you follow up with a fix? cc @scollier

Reply:
U can define filterOnlyDirectsOf ="" and it'll be ignored.

Reply:
@yanjoseph wouldn't that skip all lines?

Reply:
Oops, correct - I filed a PR to simon3z repo.